test/homework_test/tic_tac_toe_test/tic_tac_toe_test.cpp

```diff
@@ -1,7 +1,30 @@
 #define CATCH_CONFIG_MAIN  // This tells Catch to provide a main() - only do this in one cpp file
 #include "catch.hpp"
+#include "tic_tac_toe.cpp"
 
 TEST_CASE("Verify Test Configuration", "verification") {
 	REQUIRE(true == true);
 }
 
+TEST_CASE("Verify TicTacToe Game", "verification")
+
+{
+	TicTacToe game;
+
+	game.start_game("X");
+	
+	game.mark_board(1);
```

Comment:
You must call game.game_over() after every mark board and check for false for first eight and true for the ninth one.